projects/kubernetes/autoscaler/helm/sedfile.template

```diff
@@ -0,0 +1 @@
+s/version: .*$/version: ${IMAGE_TAG}/
```

Comment:
@TerryHowe what should the version of the chart be? I am running into this error while building:
```
+ cd _output/1-24/helm
+ helm package cluster-autoscaler
Error: validation: chart.metadata.version "f69e14b5d-44408693c2570873cf34960004a16bc13dc83c33" is invalid
make: *** [helm/build] Error 1
make: Leaving directory `/home/prow/go/src/github.com/aws/eks-anywhere-build-tooling/projects/kubernetes/autoscaler'
{"component":"entrypoint","error":"wrapped process failed: exit status 2","file":"k8s.io/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-09-12T21:42:08Z"}
```

Reply:
The chart needs to have a valid [semantic version](https://semver.org/) and maybe you're GIT_TAG file has that f69e14b5d in it? You are using a shasum. You can make the helm chart whatever you like, but it would probably make the most sense if it was consistent across k8s releases, don't you think? Do they use the same chart for all k8s releases `cluster-autoscaler-chart-9.21.0`? Maybe the 9.21.0 part of that?

Reply:
Tracking cluster autoscaler's chart version sounds great 👍 I'll do that. I think the weird git tag showing up is a result of combining the helm functionality with release branch functionality. I'll clean that up.